### PR TITLE
fix get_private_key route

### PR DESCRIPTION
**Pull request by timurnkey**
## Summary & Motivation
beta user found this bug 🐛 

## How I Tested These Changes
e2e

### CODE REVIEW

**Review by r-n-o — COMMENTED** [review]
You'll probably need a minor bump in `@http` when this gets released. You can create a new changeset to declare your intent (similar to https://github.com/tkhq/sdk/blob/main/.changeset/extra-http-exports.md?plain=1)

**Comment by timurnkey** [comment]
> You'll probably need a minor bump in `@http` when this gets released. You can create a new changeset to declare your intent (similar to https://github.com/tkhq/sdk/blob/main/.changeset/extra-http-exports.md?plain=1)

can you give me the ELI5 on how `.changeset` works?

**Comment by r-n-o** [comment]
> can you give me the ELI5 on how `.changeset` works?

The idea of the `.changeset` folder is to be used as a staging area for pending changesets. Each changeset is described in English, along with how it should affect the package versions when released.

For example: a bug fix will have a changeset that says "bugfix". The changeset [here](https://github.com/tkhq/sdk/blob/b409cd06790f011bf939adcf0755499b8e7497ae/.changeset/extra-http-exports.md?plain=1#L1) is a "minor" version bump because we're adding new exports.

At release time, the releaser will use the changesets in the `.changeset` folder, aggregate them, and produce:
* a changelog
* a new version for each package (e.g. if "minor bump for @http" and "bugfix bump for @http" are the two requirements, the version will be a minor bump)

So basically, creating a changeset file doesn't do anything now; think of it as instructions for the person who will release a new SDK version in the future.

**Review by r-n-o — APPROVED** [review]

**Review by andrewkmin — APPROVED** [review]
